interface/modules/zend_modules/module/Acl/view/acl/acl/index.phtml

```diff
@@ -56,39 +56,39 @@ $listener = $this->listenerObject;
 						<td><input id="add_component_section_name" type="text"></td>
 				</tr>
 				<tr>
-						<td colspan="2" style="text-align: right;"><a href="#" onclick="addSectionSave();" class="btn btn-primary" title="<?php echo $listener->z_xla('Add New Component'); ?>"><?php echo $listener->z_xlt('Add'); ?></a></td>
+						<td colspan="2" class="text-right"><a href="#" onclick="addSectionSave();" class="btn btn-primary" title="<?php echo $listener->z_xla('Add New Component'); ?>"><?php echo $listener->z_xlt('Add'); ?></a></td>
 				</tr>
 				<tr>
-						<td colspan="2" style="text-align: right;"><span id="add_component_section_message" style="color: #3AA845;"></span></td>
+						<td colspan="2" class="text-right"><span id="add_component_section_message" style="color: #3AA845;"></span></td>
 				</tr>
 		</table>
 </div>
 
-<div style="display: none;"><div id="control_div"><a href="#" ><?php echo $listener->z_xlt('Collapse All'); ?></a>|<a href="#" id="expand_tree"><?php echo $listener->z_xlt('Expand All'); ?></a></div></div>
+<div style="display: none;"><div id="control_div"><a href="#"><?php echo $listener->z_xlt('Collapse All'); ?></a>|<a href="#" id="expand_tree"><?php echo $listener->z_xlt('Expand All'); ?></a></div></div>
 
 <center>
 		<div class="scrollable default-skin" style="width: 750px; height: 600px;">
-				<table id="outer_table" align="center" >
+				<table id="outer_table" align="center">
                     <tr>
```

Comment:
use `di-none` to instead `style="display: none;"`
remove `<center>` tag.
add the Bootstrap table class.